### ISSUE TITLE
Postgres and mysql not starting by default on minv2

### ISSUE THREAD

**Issue by flakron** [Shippable/support#2361]
Up until an hour ago everything was ok, now I'm getting the below message:

```
build_ci 
     /root/e006de8c-89cb-4ce5-bb9d-9da733b587bf.sh: line 58: psql: command not found
```

**Comment by avinci** [comment]
@flakron could you give me your projectId or a link to the build. We are upgrading the back end infra and it should be transparent for users, but might be affecting you. I want to see if thats the case

**Comment by flakron** [comment]
Sure
project id: 56d45cd19d043da07b0fcabb
latest build: https://app.shippable.com/runs/56e1862800d3410c00b61da6

**Comment by avinci** [comment]
@flakron There is an issue with image picker. The new build system should pulled the old image `minv2` instead of the latest image. I am opening a ticket to get this fixed ASAP. If we need to fire a test build on your project would that be ok, i.e. once we fix the issue

**Comment by avinci** [comment]
@ric03uec can we get this fixed ASAP. The image picker should have defaulted to minv2 if the project was enabled before 2/29 and this seems to be not working as expected. Lets fix this as a CQD so that this project get unblocked.

**Comment by flakron** [comment]
Thanks for the fast response.
Sure fire the build whenever you need.

:+1:

**Comment by manishas** [comment]
@flakron I looked into this and here is what is happening-

- your subscription was upgraded to the new plan a few hours ago. 
- Since our recent release was on 2/29, all projects on the new plan and enabled on or after 2/29 do not pull shippable/minv2 as build image by default. Your project now meets this criteria, hence it started pulling u14php by default.

Can you share your shippable.yml? We can help you change it to the new format and explicitly pull minv2 and your builds should work again.

**Comment by flakron** [comment]
Sorry could not reply earlier, here is the project shippable file
```
language: php

php:
  - 5.5

before_script: mkdir -p shippable/testresults
before_script: psql -c 'create database maredoo;' -U postgres

env:
  - DB_CONNECTION=pgsql DB_DATABASE=maredoo DB_USERNAME=postgres APP_KEY=gFAPXS4RNzznTAvwhZsyiLXGmU2WdReQ

script:
  - composer self-update
  - composer update --prefer-source
  - phpunit --log-junit shippable/testresults/junit.xml

notifications:
  email:
      recipients:
         - myemail@email.com
```

**Comment by rageshkrishna** [comment]
Hi @flakron. Here's what the new YML format would look like for your project:

````
language: php

php:
  - 5.5
  
env:
  - DB_CONNECTION=pgsql DB_DATABASE=maredoo DB_USERNAME=postgres APP_KEY=gFAPXS4RNzznTAvwhZsyiLXGmU2WdReQ
  
build:
  pre_ci_boot:
    image_name: shippable/minv2
    image_tag: latest

  ci:
    - mkdir -p shippable/testresults
    - psql -c 'create database maredoo;' -U postgres
    - composer self-update
    - composer update --prefer-source
    - phpunit --log-junit shippable/testresults/junit.xml

integrations:
  notifications:
    - integrationName: "email"
      type: "email"
      recipients:
         - myemail@email.com
````

**Comment by flakron** [comment]
psql is running except it's complaining about connectivity
```
psql -c 'create database maredoo;' -U postgres
psql: could not connect to server: Connection refused
	Is the server running locally and accepting
	connections on Unix domain socket "/var/run/postgresql/.s.PGSQL.5432"?
```

**Comment by manishas** [comment]
@ric03uec we need to look into this. shippable/minv2 is being selected now, but we are not starting postgres and mysql by default as we're supposed to if the image is minv2.

**Comment by manishas** [comment]
@flakron we're going to look into our bug but you should be unblocked for your builds by adding the following in your yml:

```
services:
  - postgres
```

Let me know if that works for you. I am going to leave this issue open since we still have to track down why postgres did not start automatically for you.

**Comment by manishas** [comment]
@ric03uec here is what is happening:

- If minv2 is selected as the image by default, mysql/postgres start by default as expected
- if minv2 is explicitly selected through the pre_ci_boot section, mysql/postgres do not start even if SHIPPABLE_IS_LEGACY_IMAGE is true.

**Comment by manishas** [comment]
This is actually behaving per design. Once users move to the new yml syntax, we will not start any services by default. You can always use the services tag to start whatever you want.

Labeling this as resolved. @flakron let us know if adding the services tag to the yml as explained above doesn't work.

**Comment by manishas** [comment]
! am going to close this issue since there is nothing actionable for us to do here. 

@flakron please include the services tag in your yml and specify postgres as shown in my comment above.

**Comment by flakron** [comment]
Sorry couldn't reply earlier, I just updated the ```shippable.ym``` file. Everything works fine, now it's even a bit faster (must be related to the on-demand services).

Thanks for the help.
:-)
